Add possibility to run integration tests in parallel

jsafrane opened the pull request:
- add env. variable with etcd URL to intergration tests
- update documentation with example how to use it to find flakes

saad-ali commented on docs/devel/flaky-tests.md:
```diff
@@ -194,6 +194,41 @@ kubectl delete replicationcontroller flakecontroller
 If you do a final check for flakes with `docker ps -a`, ignore tasks that exited -1, since that's what happens when you stop the replication controller.
 
+
+# Reproducing integration test flakes
```
Title seems misleading. Contents talks about starting multiple etcd servers. It doesn't explain how this ties into the title.

jsafrane replied:
It's not only about etcd but also about starting multiple integration test processes. I updated the text a bit.

saad-ali commented:
LGTM

jsafrane commented:
rebased

jsafrane committed:
integration tests: add env. variable for etcd URL.

Many integration tests delete all keys in etcd as part of their cleanup.
To run these tests in parallel we must run several etcd daemons, each on
different port and pass etcd url to the test suite.

gmarek commented:
This change broke kubemarks. Fix in #27719
